.trunk/configs/.sqlfluff

```diff
@@ -0,0 +1,30 @@
+[sqlfluff]
+dialect = postgres
+exclude_rules = LT02, LT09
```

Comment:
The way we were replacing the SQL parameters before was insecure (literally find and replace rather than properly binding the parameters), so this helps solve a security vulnerability!

Reply:
But also adds consistency with how we format SQL files, adds linting rules, and enforces them via our Trunk pre commits.